src/memray/_memray/tracking_api.h

```diff
@@ -122,7 +122,7 @@ class NativeTrace
         return unw_backtrace((void**)data, MAX_SIZE);
     }
 
-    __attribute__((always_inline)) size_t inline exact_unwind()
```

Comment:
The reason this is here is to avoid having this particular frame in the unwinding information, so unfortunately this change cannot be done, at least just like this.

Reply:
Right, I was worried about that. Is there another way to do this?

Reply:
There are other ways to force the compiler to do this, but I have no idea if those will be invalid in `ppc64le`. We could somehow try to add a ppc64le check and disable these, but I have no idea what else will fail. If you have a ppc64le system, can you try to run the test suite with these disabled to see what else fails?